src/services/dateFnsHelper.js

```diff
@@ -0,0 +1,25 @@
+import Vue from 'vue'
+import distanceInWords from 'date-fns/distance_in_words'
+
+export default new Vue({
+  data: {
+    locale: 'en',
+    locales: {},
+    now: new Date(),
+  },
+  created () {
+    setInterval(() => { this.now = new Date() }, 10 * 1000)
+  },
+  watch: {
+    async locale (locale) {
+      if (locale === 'zh') locale = 'zh_cn' // https://date-fns.org/v1.29.0/docs/I18n
```

Comment:
Is zh the only local that needs this or can we map the locales more generally?

Reply:
Yes, is the only one. I switched it to traditional chinese now actually, that seems to be the preferred version in Taiwan according to the internet.

Reply:
Isn't there something like cn_TW?

Reply:
It's using `zh_tw'` now,  which is called "Chinese Traditional" by date-fns.

> The Simplified is the standard writing form employed in the mainland of China and the Traditional form is mainly used in Taiwan and Hong Kong. Although the majority of the population uses Simplified Chinese there is a trend towards returning to Traditional Chinese. 
says http://www.iwtservices.com/resources/chinese_sort.htm

It probably means we should account for these locales with regions at some point...